packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -265,22 +265,21 @@ const PRESETS = [
             type: TEMPLATE_SOURCE_BUNDLE,
             id: 'chakra-storefront'
         },
-        getQuestions: () =>
-            createRetailReactAppQuestions({
-                'project.hybrid': false,
-                'project.name': 'demo-storefront',
-                'project.commerce.instanceUrl': 'https://zzte-053.dx.commercecloud.salesforce.com',
-                'project.commerce.clientId': '1d763261-6522-4913-9d52-5d947d3b94c4',
-                'project.commerce.siteId': 'RefArch',
-                'project.commerce.organizationId': 'f_ecom_zzte_053',
-                'project.commerce.shortCode': 'kv7kzm78',
-                'project.commerce.isSlasPrivate': false,
-                'project.einstein.clientId': '1ea06c6e-c936-4324-bcf0-fada93f83bb1',
-                'project.einstein.siteId': 'aaij-MobileFirst',
-                'project.dataCloud.appSourceId': 'fb81edab-24c6-4b40-8684-b67334dfdf32',
-                'project.dataCloud.tenantId': 'mmyw8zrxhfsg09lfmzrd1zjqmg',
-                'project.demo.enableDemoSettings': false
-            }),
+        answers: {
```

Comment:
It turns out that most of the presets are broken too. But I've now fixed them.

We recently introduced `interactive: true` in the `next` branch. Those presets that are interactive will prompts the users some questions, while non-interactive presets won't ask the questions but will silently use the given answers. 

So the bug fix is making sure that non-interactive presets like this one have the `answers`.